lib/sync/sync-server.js

```diff
@@ -45,10 +45,16 @@ var syncWorker;
 var DEFAULT_SYNC_CONF = {
   /** @type {Number} how often pending workers should check for the next job, in ms. Default: 500 */
   pendingWorkerInterval: 500,
+  /** @type {Object} the backoff strategy for the pending worker to use. Default is exponetial, and the max delay is 60s */
+  pendingWorkerBackoff: {strategy: 'exp', max: 60*1000},
   /** @type {Number} how often ack workers should check for the next job, in ms. Default: 500 */
   ackWorkerInterval: 500,
+  /** @type {Object} the backoff strategy for the ack worker to use. Default is exponetial, and the max delay is 60s */
```

Comment:
We should inlcude some info on what the different strategies are, and what exactly 'max delay' means